### DISCUSSION TITLE
Form: Only show one error

### DISCUSSION THREAD

**Discussion by Archisketch-Finn** [ant-design/ant-design#48412] [Q&A] [answered]
Is there a way to show only the first error of the rules without using validator() to check it manually?

![image](https://github.com/ant-design/ant-design/assets/105255382/8937dcea-90d1-4379-bc2e-2fb961803774)

![image](https://github.com/ant-design/ant-design/assets/105255382/e47540e8-267f-4f6a-9eaa-4bd0a510d9ba)

**Comment by zombieJ** [comment, marked as answer]
Use `validateFirst` to fail immediately for any rule match.

**Reply by Archisketch-Finn** [reply]
Thank bro 🙏
